core/src/main/java/com/linecorp/armeria/unsafe/PooledObjects.java

```diff
@@ -143,6 +144,37 @@ public static void close(Object obj) {
                 }
             }
         }
+        if (obj instanceof WebSocketFrame) {
```

Comment:
Any chance where `obj` is both `HttpData` and `WebSocketFrame`?

Reply:
I don't believe so unless a user maliciously writes a class that implements those interfaces.
Even so, an `HttpData` should have all the information in its `array()` and the case is already handled above.